packages/assert/src/types.js

```diff
@@ -173,16 +173,16 @@
  * ```
  *
  * The normal convention is to locally rename `quote` to `q` and
- * `details` to `d`
+ * `details` to `X`
  * ```js
- * const { details: d, quote: q } = assert;
+ * const { details: X, quote: q } = assert;
```

Comment:
```suggestion
 * import { assert, details as X, q } from '@agoric/assert';
```

The text above should be changed to 

```
 * The normal convention is to locally rename details to `X` and import 'q' and 'assert' unmodified.
```

Reply:
Thanks